vassal-app/src/main/java/VASSAL/build/module/BasicLogger.java

```diff
@@ -114,6 +118,22 @@ public BasicLogger() {
     newLogAction.setEnabled(false);
     logInput = new ArrayList<>();
     logOutput = new ArrayList<>();
+    startAutosave();
+  }
+
+  private void startAutosave() {
+    scheduler.scheduleAtFixedRate(this::autosave, 5, 5, TimeUnit.SECONDS);
```

Comment:
Five seconds is probably too short.

There's nothing to debounce the autosave if autosaving takes longer than the autosave interval.

Reply:
I changed that to start after a minute and save every 5 minutes.